arch/arm/arm64/include/uk/asm/lcpu.h

```diff
@@ -47,25 +46,25 @@
  */
 struct __regs {
 	/* General-purpose registers, from x0 ~ x29 */
-	uint64_t x[30];
+	unsigned long x[30];
```

Comment:
As this is not portable code, but architecture specific it would be better to use fixed sizes. I would instead replace uint64_t with _u64 etc rather than using generic types.